lib/src/eager_span_scanner.dart

```diff
@@ -99,12 +99,12 @@ class EagerSpanScanner extends SpanScanner {
   bool scan(Pattern pattern) {
     if (!super.scan(pattern)) return false;
 
-    var newlines = _newlinesIn(lastMatch![0]);
+    var newlines = _newlinesIn((lastMatch![0])!);
```

Comment:
It might be worth declaring a local variable for `lastMatch![0]!` so you don't have to repeat that.

Reply:
done